src/main/java/de/metas/ui/web/handlingunits/process/WebuiHUTransformCommand.java

```diff
@@ -217,7 +211,7 @@ public final WebuiHUTransformCommandResult execute()
 			}
 			case LU_Set_Ownership:
 			{
-				return action_updatePlanningReceiptOwnerPM(row, parameters.isHuPlanningReceiptOwnerPM_LU());
+				return action_updatePlanningReceiptOwnerPM(row, true);
```

Comment:
We can't leave the parameter value dangling and ignore it.
Like, what if someone wants to tell the system that a Pallete with ReceiptOwnerPM=Y is **not** the businespartner's property.

So, if you are sure this will never happen (which i seriously doubt, please remove the whole `HuPlanningReceiptOwnerPM_LU` field
Otherwise, please debug why tf it's not `true` when it should be and fix that.